src/com/simulator/controller/SmartHomeSimulatorController.java

```diff
@@ -326,6 +332,32 @@ public void setUsersInLayout(){
         }
     }
 
+     /**
+     * Sets AC/Heating Icons in the house layout view.
+     */
+    @FXML
+    public void setTemperatureIconsInLayout(){
+        List<Room> rooms = house.getRooms();
+        for (int loop=0; loop < rooms.size();  loop++)
```

Comment:
I would change this for loop to for(Room room : rooms) it will loop over each element in the array and you will be able to directly access the room makes for cleaner code

Reply:
So everywhere that you use rooms.get(loop) you could replace with room

Reply:
Great suggestion, it's done!